### PR TITLE
Use IsoCodes test data for validators

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Waiting https://github.com/ronanguilloux/IsoCodes/pull/92 to be finished.
- [x] Use IsoCodes data tests
- [x] Remove `composer.json` hack
- [x] Manage special cases like `IP` or `Isbn`...
- [ ] ~Update "How to add a constraint documentation" doc accordingly.~ Documentation does not exist yet.

If https://github.com/ronanguilloux/IsoCodes/pull/93 is merged before new stable release of IsoCodes, remove special case from `CreditCardValidatorTest::getValidValues`.

### CODE REVIEW

File: tests/Constraints/AbstractConstraintValidatorTest.php [hunk 1]
```diff
@@ -103,7 +110,52 @@ protected function createConstraint()
...
+
+        return $values;
+    }
+
+    /**
+     * @return AbstractIsoCodeTest
+     */
+    private function getIsoCodesTestInstance()
+    {
+        // Test classes seems not be autoloaded by default.
```
Comment (greg0ire) [inline tests/Constraints/AbstractConstraintValidatorTest.php]: Configure `autoload-dev`, maybe?

Comment (Soullivaneuh) [inline tests/Constraints/AbstractConstraintValidatorTest.php]: What do you mean?

Comment (greg0ire) [inline tests/Constraints/AbstractConstraintValidatorTest.php]: In `composer.json`, you can specify how to generate autoload mechanisms for whatever you want (including those test classes I think). Using `autoload-dev` will ensure you don't clutter your production autoload mechanism.

Comment (Soullivaneuh) [inline tests/Constraints/AbstractConstraintValidatorTest.php]: Thank you! :heart:

**Pull request merged**